docs/operations/metrics.md

```diff
@@ -256,6 +256,8 @@ These metrics are for the Druid Coordinator and are reset each time the Coordina
 |`interval/skipCompact/count`|Total number of intervals of this datasource that are skipped (not eligible for auto compaction) by the auto compaction.|datasource.|Varies.|
 |`coordinator/time`|Approximate Coordinator duty runtime in milliseconds. The duty dimension is the string alias of the Duty that is being run.|duty.|Varies.|
 |`coordinator/global/time`|Approximate runtime of a full coordination cycle in milliseconds. The `dutyGroup` dimension indicates what type of coordination this run was. i.e. Historical Management vs Indexing|`dutyGroup`|Varies.|
+|`metadata/kill/audit/count`|Total number of audit logs deleted from metadata store audit table.| |Varies.|
```

Comment:
The metric can give the user an idea of how many audit logs are being automatically deleted. Basically how many audit logs do they have within the duration configured. This can help them get a sense if they want to increase or decrease the durationToRetain. I think we should also have a metric for the current size (in rows) of the audit logs. Both of these metrics can be used together. You will be able to compare the current size (in rows) vs the kill size per run (in rows) and see if you want to increase or decrease durationToRetain. 

I will add  a metric for the current size (in rows) of the audit logs in a separate PR

Reply:
> The metric can give the user an idea of how many audit logs are being automatically deleted. Basically how many audit logs do they have within the duration configured. This can help them get a sense if they want to increase or decrease the durationToRetain. 

Thanks for the explanation. Could you please explain this in the doc as well? Also please mention that this metric is emitted only when `druid.coordinator.kill.audit.on` is set to true.